### PR TITLE
No need for the --disable-setuid-sandbox flag

### PR DESCRIPTION
**Pull request by glaforge**
It seems the --disable-setuid-sandbox flag is becoming deprecating and is being removed from the Chrome codebase.
I tried running the sample without this flag, and the screenshots are properly generated.

/cc @steren

### CODE REVIEW

**Comment by steren** [comment]
Thanks!

File: appengine/headless-chrome/app.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ app.use(async (req, res) => {
 
   // [START browser]
   const browser = await puppeteer.launch({
-    args: ['--no-sandbox', '--disable-setuid-sandbox']
+    args: ['--no-sandbox']
```
Comment (MylesBorins) [inline appengine/headless-chrome/app.js]: has this been confirmed in a deplyoment? If I recall the `setuid-sandbox` was necessary on standard

Comment (fhinkel) [reply appengine/headless-chrome/app.js]: Tests are green with and without the flag. @steren, how much confidence do you have in your test? :wink:

Comment (fhinkel) [reply appengine/headless-chrome/app.js]: Ran it manually on standard. Works fine without the flag.

**Pull request merged**